GUIDELINES.md

```diff
@@ -154,12 +160,9 @@ At this point, we eschew any Node.js version below 4.x. This means, that usage o
 features is possible. And there are some good features to take advatage of. Here are
 some of our recommendations.
 
-* Use `let` and `const` instead of `var`. There is rarely ever a valid case for using
-  `var` in modern Javascript code. If the variable is immutable (which we believe most
-  should be), use `const`. If the variable may be written, use `let`.
```

Comment:
Looks good overall. I wonder if there is any value to leaving this text in the document some place. I realize it's not relevant in the "what linting won't catch" section any longer. Maybe just as a comment in the rules above?